gradlew.bat

```diff
@@ -0,0 +1,84 @@
+@if "%DEBUG%" == "" @echo off

```

Comment:
are we supporting running heroic on windows now?

Reply:
Nah, its autogenerated by `gradle init`.

Reply:
is there value in having this file committed then?